src/api/app/controllers/source_project_controller.rb

```diff
@@ -1,4 +1,5 @@
 class SourceProjectController < SourceController
+  rescue_from Pundit::NotAuthorizedError, with: :handle_pundit_not_authorized
```

Comment:
This already happens in `ApplicationController`

Reply:
I have to do it locally to generate a specific error message to match the error message expected by the API..